zeppelin-interpreter/src/main/thrift/RemoteInterpreterService.thrift

```diff
@@ -63,6 +65,35 @@ struct RemoteApplicationResult {
 }
 
 /*
+  remote interpreter process --> request --> zeppelin server
+ */
+enum RemoteZeppelinServerControlEvent {
+  REQ_RESOURCE_PARAGRAPH_RUN_CONTEXT = 1,
+  RES_RESOURCE_PARAGRAPH_RUN_CONTEXT = 2
+}
+
+enum RemoteZeppelinServerResourceType {
+  RESOURCE_PARAGRAPH_RUN_CONTEXT = 1
+}
+
+struct ZeppelinServerResourceParagraphRunner {
+  1: string noteId,
+  2: string paragraphId
+}
+
+struct ZeppelinServerResource {
+  1: RemoteZeppelinServerResourceType type,
+  2: string eventOwnerKey,
+  3: string msg
+}
+
+struct RemoteZeppelinServerController {
+  1: RemoteZeppelinServerControlEvent type,
+  2: string eventOwnerKey
+  3: string msg
+}
```

Comment:
basically, we don't need `enum RemoteZeppelinServerControlEvent`, `enum RemoteZeppelinServerResourceType`, `struct ZeppelinServerResourceParagraphRunner`, `struct ZeppelinServerResource`, `struct RemoteZeppelinServerController`.

Please try use generic message data structure `RemoteInterpreterEvent` for sending event from interpreter process to zeppelin server process.